### PR TITLE
[enhancement](log4j) support high performance mode for log4j2 to escape potential bottleneck for doris read and write

### PR DESCRIPTION
**Pull request by caiconghui**
## Proposed changes

Issue Number: close #xxx

As we know, log4j2 some times may be bottleneck in doris fe when there are many logs to be output in sync mode while asynchronous logging has a better performance， and we find that capturing caller location has a similar impact across all logging libraries, and slows down asynchronous logging by about 30-100x. so, here we provide three log mode for log4j2 to meet the needs of different users.
refer to https://logging.apache.org/log4j/2.x/performance.html

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by caiconghui** [comment]
run buildall

**Review by weizuo93 — APPROVED** [review]
LGTM

File: conf/fe.conf [hunk 1]
```diff
@@ -38,6 +38,9 @@ JAVA_OPTS_FOR_JDK_9="-Xss4m -Xmx8192m -XX:SurvivorRatio=8 -XX:MaxTenuringThresho
 # INFO, WARN, ERROR, FATAL
 sys_log_level = INFO
 
+# NORMAL, BRIEF, ASYNC
+sys_log_mode = NORMAL
```
Comment (yiguolei) [inline conf/fe.conf:42]: comment this line and set normal as default value

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**